modules/core/src/main/java/org/apache/ignite/internal/processors/authentication/IgniteAuthenticationProcessor.java

```diff
@@ -136,8 +161,8 @@
     /** Node activate future. */
     private final GridFutureAdapter<Void> activateFut = new GridFutureAdapter<>();
 
-    /** Validate error. */
-    private String validateErr;
+    /** User management operations. */
+    private final EnumSet<SecurityPermission> userOps = EnumSet.of(CREATE_USER, DROP_USER, ALTER_USER);
```

Comment:
This set used only one time so let's remove it and use explicit check like 

`if (op == CREATE_USER || op == ALTER_USER || op == DROP_USER)`

Reply:
Done.